### ISSUE TITLE
Doubt

### ISSUE THREAD

**Issue by linzhi123** [kendricktan/drawlikebobross#1]
self.conv = nn.Sequential(
            # Input: (channel_num) x 256 x 256
            nn.Conv2d(channel_num, filter_num, 4, 2, 1, bias=False),
            nn.LeakyReLU(0.2, inplace=True),
            # (filter_num) x 128 x 128
            nn.Conv2d(filter_num, filter_num * 2, 4, 2, 1, bias=False),
            nn.BatchNorm2d(filter_num * 2),
            nn.LeakyReLU(0.2, inplace=True),
            # (filter_num * 2) x 64 x 64
            nn.Conv2d(filter_num * 2, filter_num * 2, 8, 4, 1, bias=False),
            nn.BatchNorm2d(filter_num * 2),
            nn.LeakyReLU(0.2, inplace=True),
            # (filter_num * 2) x 16 x 16
            nn.Conv2d(filter_num * 2, filter_num, 8, 4, 1, bias=False),
            nn.BatchNorm2d(filter_num),
            nn.LeakyReLU(0.2, inplace=True),
            # (filter_num) x 3 x 3
        )

Can you tell me how these parameters are set, if I modify the size of the input of the picture?
